emailsender/cmd/app/main.go

```diff
@@ -39,9 +40,19 @@ func main() {
 
 	ctx := context.Background()
 
+	// initialize components
+	sesClient, err := email.NewSES(ctx)
+	if err != nil {
+		glog.Errorf("Failed to initialise SES Client: %v", err)
+		os.Exit(1)
+	}
+	temporarySenderName := "sender@example.com"
```

Comment:
This should be a config value set per environment we're in, in Prod in will be:
```suggestion
	temporarySenderName := "noreply@mail.acs.rhcloud.com"
```